src/winston.classes.ts

```diff
@@ -11,23 +11,38 @@ export class WinstonLogger implements LoggerService {
     this.context = context;
   }
 
-  public log(message: any, context?: string) {
-    return this.logger.info(message, { context: context || this.context });
+  public log(message: any, meta?: string | Record<string, unknown>) {
+    if('object' === typeof meta) {
```

Comment:
`context?: string` is now `meta?: string | Record<string, unknown>`.
Checking the type of `meta` for backward compatibility.

Reply:
This might be better.
```typescript
public log(message: any, context?: string, meta?: Record<string, unknown>) {}
```

Reply:
hey @mizozobu thanks for the contribution,
If I am using [LoggerService for bootstraping](https://github.com/gremo/nest-winston#use-as-the-main-nest-logger-also-for-bootstrapping), would I have problems if I don't implement the same LoggerService interface https://github.com/nestjs/nest/blob/master/packages/common/services/logger.service.ts#L11-L17?

I import using the following way
```typescript
import {  Logger, LoggerService } from '@nestjs/common';
```
is it possible to use like this?
```typescript
//with loggerService
logger.log({
   "message":"test",
   "metadata":{
      "test1":"test",
      "test2":"test"
   }
})
```

```typescript
//winston
this.logger.info(message.message, { context: this.context, ...message.metadata });
```